documentation/src/main/asciidoc/reference/coordination.asciidoc

```diff
@@ -0,0 +1,508 @@
+[[coordination]]
+= Coordination
+
+include::components/coordination-comparison-note.asciidoc[]
+
+[[coordination-basics]]
+== Basics
+
+[NOTE]
+====
+Coordination is a complex topic;
+for a gentle introduction to the problems it solves,
+have a look at the <<architecture-examples>>.
+====
+
+An application using Hibernate Search usually relies on multiple threads,
+or even multiple application instances,
+which will update the database concurrently.
+
+The coordination strategy defines how these threads/nodes will coordinate with each other
+in order to update indexes according to these database updates,
+in a way that ensures consistency,
+prevents data loss,
+and optimizes performance.
+
+This strategy is set through configuration properties:
+
+[source]
+----
+hibernate.search.coordination.strategy = database-polling
+----
+
+The default for this property is `none`.
+
+See the following subsections for details about available strategies.
+
+[[coordination-none]]
+== No coordination
+
+[[coordination-none-basics]]
+=== Basics
+
+The default strategy is the simplest and does not involve any additional infrastructure
+for communication between application nodes.
+
+All <<mapper-orm-indexing-automatic,automatic indexing>> operations
+are executed directly in application threads,
+which gives this strategy the unique ability to provide <<mapper-orm-indexing-automatic-synchronization,synchronous indexing>>,
+at the cost of a few limitations:
+
+* <<limitations-parallel-embedded-update>>
+* <<limitations-backend-indexing-error>>
+
+include::components/coordination-comparison-note.asciidoc[]
```

Comment:
We have several links directly to this section, and I don't think people will bother to go back to the top of the section when the land here.

So I'd rather repeat this, since it's quite short and its only purpose is to give more general information about the coordination strategies.